operations/mimir/etcd.libsonnet

```diff
@@ -2,7 +2,7 @@ local etcd_cluster = import 'etcd-operator/etcd-cluster.libsonnet';
 
 etcd_cluster {
   etcd:
-    $.etcd_cluster('etcd', env=[{
+    $.etcd_cluster('etcd', size=3, env=[{
```

Comment:
`size=3` doesn't do any increase.

What do you think about adding a new `_config` field to specify the etcd cluster size?

Reply:
Ugh, my bad :-) I'll be happy to add new `_config` entry as well, that might be even easier to manage in the long run.

Reply:
Fixed; I changed this to be a PR to only allow overriding the default size.